systemtest/src/test/java/io/strimzi/systemtest/components/kafka/KafkaST.java

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest;
+package io.strimzi.systemtest.components.kafka;
```

Comment:
I do like that you've put this in the kafka package, but I think that components here is just redundant...

Reply:
Ye, that sounds reasonable to me